packages/table/src/common/_variables.scss

```diff
@@ -33,6 +33,7 @@ Color aliases
 
 $table-background-color: $light-gray5 !default;
 $dark-table-background-color: $dark-gray4 !default;
+$input-placeholder-color: $pt-text-color-muted !default;
```

Comment:
I don't think you need this variable, especially since it's the same name as another (non-public) variable in core/src/components/forms. You can just reference the `$pt-text-color-muted` and `$pt-dark-text-color-muted` variables directly where you need the colors.